android/app/build.gradle

```diff
@@ -51,33 +57,54 @@ android {
 
     signingConfigs {
         release {
-//      Commented to build without release keys
-//            keyAlias keystoreProperties['keyAlias']
-//            keyPassword keystoreProperties['keyPassword']
-//            storeFile file(keystoreProperties['storeFile'])
-//            storePassword keystoreProperties['storePassword']
+            keyAlias keystoreProperties['keyAlias']
+            keyPassword keystoreProperties['keyPassword']
+            storeFile file(keystoreProperties['storeFile'])
+            storePassword keystoreProperties['storePassword']
         }
     }
 
     buildTypes {
-        release {
-            // To release we need to switch to signingConfigs.release
+        debug {
+            // default
             signingConfig signingConfigs.debug
+        }
+
+        release {
+            // add --release to run command
```

Comment:
Which command is this? flutter or gradle?

Reply:
Flutter, through terminal or IDE.